articles/multifactor-authentication/google-auth/admin-guide.md

```diff
@@ -0,0 +1,116 @@
+# Google Authenticator for Administrators
+
+::: panel-info Note:
+This page refers to using Google Authenticator instead of Auth0 Guardian. Google Authenticator can also be used with Guardian(your users choose which to use) when **Push Notifications** is enabled. [Click here for more information on Guardian.](/multifactor-authentication/guardian)
+:::
+
+## Enabling Google Authenticator for MFA
+
+To turn on Google Authenticator for two-step verification, first visit the [Multifactor Auth](${uiURL}/#/guardian) page from the dashboard. Then click on the link to use a different provider.
+
+![](/media/articles/mfa/change-provider.png)
+
+Then you can use the slider to turn on Google Authenticator.
+
+![](/media/articles/mfa/toggle-google-auth.png)
+
+## Google Authenticator Supported Devices
+
+Your users must have a supported device to use the Google Authenticator app. If some of your users have an unsupported device type, they may be able to use Auth0's Guardian app instead of Google Authenticator. [Click here for information on using Guardian.](/multifactor-authentication/guardian/admin-guide)
+
+| **OS** | **Google Authenticator** | 
+| --- | --- | 
+| **iOS** | Requires iOS 5.0 or later |
+| **Android** | Requires Android version 2.1 or later |
+| **Windows** | Unsupported |
```

Comment:
A good example of using a code generator app. Will have to try but I think that Microsoft Authenticator is compatible with Google Authenticator provider